gef.py

```diff
@@ -2789,7 +2792,7 @@ def gef_on_new_unhook(func): return gdb.events.new_objfile.disconnect(func)
 #
 
 class PieVirtualBreakpoint(object):
-    """PIE virtual breakpoint (not real breakpoint)"""
+    """PIE virtual breakpoint (not real breakpoint)."""
```

Comment:
Hey @Escapingbug. First of, thanks for your contrib. Always appreciated :+1: 

Here can you explain why not use the [`gdb.Breakpoint`](https://sourceware.org/gdb/current/onlinedocs/gdb/Breakpoints-In-Python.html) class as a parent to `PieVirtualBreakpoint`. It really looks like it could make the result lighter and more pythonic, but maybe you had a reason to do so.

Reply:
Thanks. This is because this "breakpoint" is not actually a gdb Breakpoint. That's why I call it a "virtual breakpoint" instead a "breakpoint". This class meant to represent a virtual breakpoint instead a real breakpoint in GDB. 

When we  create a new virtual breakpoint, nothing is actually done in GDB. What we do is create a gdb breakpoint when we start of attach to a process, but not when initialize. If it is a real breakpoint, aka `gdb.breakpint`, it creates a new breakpoint while initialization. This is what  `__init__` does in `gdb.breakpoint`, so I think these two things differ.